1.9/administration/installing/custom/remove-a-node.md

```diff
@@ -0,0 +1,66 @@
+---
+post_title: Removing Agent Nodes
+nav_title: Remove Node
+menu_order: 801
+---
+
+You can remove agent nodes from an active DC/OS cluster. 
+
+When you change Mesos attributes (`⁠⁠⁠⁠/var/lib/dcos/mesos-slave-common`⁠⁠⁠⁠) or resources (⁠⁠⁠⁠`/var/lib/dcos/mesos-resources`⁠⁠⁠⁠), you have to remove the agent node and re-register it with the master under a new UUID. The master will then recognize the new attributes and resources specification.
+
+### Prerequisites:
+
+*   DC/OS is installed using the [custom](/docs/1.9/administration/installing/custom/) installation method.
```

Comment:
Yeah.  The only prerequisite is that you can access the Adminrouter endpoints (and are authenticated/authorized).